### PR TITLE
Both free-tier mode and unlimited

### PR DESCRIPTION
**Pull request by bakulf**

### CODE REVIEW

File: management-src/page.js [hunk 1]
```diff
@@ -52,10 +52,22 @@ class Page {
...
 
-    const migrationCompleted = document.getElementById("migrationCompleted");
-    migrationCompleted.checked = config.migrationCompleted || false;
-    migrationCompleted.onchange = _ => {
-      browser.runtime.sendMessage({ type: "setMigrationCompleted", value: migrationCompleted.checked });
+    const reminderEnabled = document.getElementById("reminderEnabled");
+    reminderEnabled.checked = config.reminder;
+    reminderEnabled.onchange = _ => {
+      browser.runtime.sendMessage({ type: "setReminder", value: reminderEnabled.checked });
+    }
```
Comment (jonathanKingston) [inline management-src/page.js]: Nit: Semi colon.
Does all of this pass linting?

Comment (bakulf) [reply management-src/page.js]: Yes it does. Note that we have the same code in many other places.

File: src/popup/popup.html [hunk 2]
```diff
@@ -20,10 +20,11 @@ <h1 id="introHeading"></h1>
       <div class="enabled-circle" id="enabled-circle-3"></div>
       <div class="enabled-circle" id="enabled-circle-4"></div>
       <button id="stateButton"></button>
+      <span id="passCountdown" hidden></span>
     </div>
     <div id="toggleRow" hidden>
       <span data-l10n="popupToggleButton"></span>
-      <input type="checkbox" id="toggleButton" />
+      <input type="checkbox" class="toggleButton" id="toggleButton" />
```
Comment (jonathanKingston) [inline src/popup/popup.html]: Can we rename this id to reduce confusion in future?

File: src/popup/view.js [hunk 3]
```diff
@@ -72,6 +75,11 @@ export class View {
...
   }
 
+  static syncShowPassCountdown(state) {
+    const countdown = document.getElementById("passCountdown");
+    countdown.hidden = !state;
+  }
+
   static hideToggleButton() {
     let toggleRow = document.getElementById("toggleRow");
     toggleRow.setAttribute("hidden", "hidden");
```
Comment (jonathanKingston) [inline src/popup/view.js]: Aside: This should probably use `toggleRow.toggleAttribute("hidden", true)`

File: src/commons/utils.js [hunk 4]
```diff
@@ -128,6 +130,24 @@ const ConfigUtils = {
     return await this.getStorageKey("passesTimeout") || DEFAULT_PASSES_TIMEOUT;
   },
 
+  async setReminder(reminder) {
```
Comment (jonathanKingston) [inline src/commons/utils.js]: Why is this block duplicated with storageUtils? Can one call the other instead? Feel free to make this a follow up though.

Comment (bakulf) [reply src/commons/utils.js]: hehe this code is shared with the popup where we don't have storageUtils. You wrote this code in the first place in this way :)

**Pull request merged**